lib/galaxy/files/templates/examples/production_elabftw.yaml

```diff
@@ -0,0 +1,27 @@
+- id: elabftw
+  version: 0
+  name: eLabFTW
+  description: |
+    eLabFTW is a free and open source electronic lab notebook from Deltablot. It can keep track of experiments,
+    equipment, and materials from a research lab. Each lab can either host their own installation or go for Deltablot's
+    hosted solution. This template configuration allows you to connect to an eLabFTW instance of your choice.
+  variables:
+    endpoint:
+      label: eLabFTW instance endpoint (e.g. https://demo.elabftw.net)
+      type: string
+      help: |
+        The endpoint of the eLabFTW server you are connecting to. This should be the full URL including the protocol
+        (http or https) and the domain name.
+  secrets:
+    api_key:
+      label: API Key
+      help: |
+        The API key to use to connect to the eLabFTW server. Navigate to the _Settings_ page on your eLabFTW server and
+        go to the _API Keys_ tab to generate a new key. Choose "Read/Write" permissions to enable both importing and
+        exporting data. "Read Only" API keys still work for importing data to Galaxy, but they will cause Galaxy to
+        error out when exporting data to eLabFTW.
+  configuration:
+    type: elabftw
+    endpoint: "{{ variables.endpoint }}"
+    api_key: "{{ secrets.api_key }}"
+    writable: true
```

Comment:
Should we make it an option? Maybe for some use-cases, you don't want to accidentally write into it?

Reply:
It may be confusing that people are asked twice if they want to write into it, because the eLabFTW UI already asks you this when generating the key

![grafik](https://github.com/user-attachments/assets/83fed7b9-94e3-4e34-ae67-ae3d64bef38b)

however, that's still problematic because if you use a read-only key, they the file source is listed as writable and that's also confusing.

In other words, we'gotta pick our poison, so indeed we can make it an option.

Reply:
The "good solution" is retrieving the API keys via the API

![grafik](https://github.com/user-attachments/assets/005a0b6b-b794-4edb-addb-9ac99fe158fb)

and compare the hash to know if it's read-only or not. Using that information let the file source adjust its `writable` property itself. But that may not be worth the effort, because it involves digging into eLabFTW's code to find out what the hashing algorithm is.

Reply:
Done in [c4693d4](https://github.com/galaxyproject/galaxy/pull/19493/commits/c4693d42448552ea1e7ae581f8240c4884f4d350)